### PR TITLE
React v. 16.4.1, ES6 syntax, Webpack

### PR DESCRIPTION
**Pull request by nwelchr**
Please let me know if you have any questions. Enjoy!

### CODE REVIEW

File: examples/react-es6/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# React TodoMVC Example
```
Comment (FadySamirSadek) [inline examples/react-es6/readme.md]: Please update the readme file to contain data about your example, for example, add links and resources to webpack and ES6 also edit the title to show that this example contains a bundled version of react

File: examples/react-es6/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "react-router-dom": "^4.3.1",
+    "todomvc-app-css": "^2.0.0",
+    "todomvc-common": "^1.0.1",
+    "webpack": "^3.9.0"
+  },
+  "devDependencies": {
+    "bin-up": "^1.1.0"
+  },
+  "scripts": {
+    "cy:open": "bin-up cypress open --project ../.. --env framework=react",
```
Comment (FadySamirSadek) [inline examples/react-es6/package.json]: Please edit this line to contain the right example name.

File: examples/react-es6/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "cy:run": "bin-up cypress run --project ../.. --env framework=react",
```
Comment (FadySamirSadek) [inline examples/react-es6/package.json]: same here

**Comment by nwelchr** [comment]
Thanks for the feedback! I made a new commit with the changes you asked for. Let me know if it's ok! (This is my first time making a pull request so I hope I'm doing this right)

File: examples/react-es6/src/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+node_modules
```
[the remark this answers is not in the document]
Comment (nwelchr) [reply examples/react-es6/src/.gitignore]: Didn't need this, was just me being silly haha

**Comment by nwelchr** [comment]
Hello! I can't seem to understand why checks have failed. It seems that they are all happening with files I did not modify. I did everything you said so all should be working fine. I would really appreciate if you could advise on this. Thank you!

**Comment by Fruitseller** [comment]
Hey @nwelchr.
I'm testing your PR and I have some difficulties.
I run `npm install` in your folder, started the server with `npm run server` in the root directory and wanted to run the testsuite with `npm run cy:run` in the react-es6 folder but the test failed with the error that the app has not started yet. :confused:

I also wanted to start your app locally but failed at that. I opened `http://localhost:8000/examples/react-es6/` and it seems all files loaded correctly but the react app has not started. I see only the HTML. :weary:
I hope you can help me out. :blush::blush:

**Comment by nwelchr** [comment]
That's really interesting... Thanks for letting me know! I've been testing just by running the server from the examples/react-es6 directory and loading `http://localhost:8000`, and that works perfectly, but for some reason when I run the server from another folder and try to access it, it breaks as you described. I'll try to see what the issue is, but would you happen to know why that might be the case?

**Pull request closed without merging**